lesson1/src/stacks_queues/stackProblems.js

```diff
@@ -22,9 +22,12 @@ class StackProblems {
     for (let i = 0; i < string.length; i++) {
       let char = string.charAt(i);
       // todo: do you'r magic ! 🙌
+      if (char === '(' || char === ')') {
```

Comment:
🤔 this is a nice attempt but `hello)(` should return false (parenthesis are not balanced) and in your case will return true ... I also think `hello((` would return true in your case ... and that one is easier to tell its unbalanced 😝